modules/rosa-deleting-cluster.adoc

```diff
@@ -9,24 +9,109 @@ endif::[]
 
 :_content-type: PROCEDURE
 [id="rosa-deleting-cluster_{context}"]
-= Deleting a cluster
+ifndef::sts[]
+= Deleting a ROSA cluster
+endif::sts[]
+ifdef::sts[]
+= Deleting a ROSA cluster and the cluster-specific IAM resources
+endif::sts[]
 
-You can delete an {product-title} cluster using the `rosa` CLI.
+ifndef::sts[]
+You can delete a {product-title} cluster using the ROSA CLI (`rosa`).
+endif::sts[]
 
 ifdef::sts[]
-You can also use the `rosa` CLI to delete the AWS Identity and Access Management (IAM) account-wide roles, the cluster-specific Operator roles, and the OpenID Connect (OIDC) provider. The cluster deletion must complete before you remove the IAM resources, because the resources are used in the cluster deletion and clean-up processes.
+You can delete a {product-title} (ROSA) with AWS Security Token Service (STS) cluster by using the ROSA CLI (`rosa`) or {cluster-manager-first}.
 
-If add-ons are installed, the deletion takes longer because add-ons are uninstalled before the cluster is deleted. The amount of time depends on the number and size of the add-ons.
+After deleting the cluster, you can clean up the cluster-specific Identity and Access Management (IAM) resources in your AWS account by using the ROSA CLI (`rosa`). The cluster-specific resources include the Operator roles and the OpenID Connect (OIDC) provider.
 
 [IMPORTANT]
 ====
-Account-wide IAM roles and policies might be used by other ROSA clusters in the same AWS account. You must only remove the resources if they are not required by other clusters.
+The cluster deletion must complete before you remove the IAM resources, because the resources are used in the cluster deletion and clean-up processes.
 ====
+
+If add-ons are installed, the cluster deletion takes longer because add-ons are uninstalled before the cluster is deleted. The amount of time depends on the number and size of the add-ons.
 endif::sts[]
 
+.Prerequisites
+
+* You have installed a ROSA cluster.
+* You have installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your installation host.
+
 .Procedure
 
+ifdef::sts[]
+. Obtain the cluster ID, the Amazon Resource Names (ARNs) for the cluster-specific Operator roles and the endpoint URL for the OIDC provider:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> <1>
```

Comment:
We could offer this command:
`rosa describe cluster --cluster=<cluster_name> |egrep '^ID:|OIDC|-openshift-'`
which provides
```
ID:                         <cluster-id>
 - arn:aws:iam::123456789012:role/<cluster>-a8m7-openshift-machine-api-aws-cloud-credentials
 - arn:aws:iam::123456789012:role/<cluster>-a8m7-openshift-cloud-credential-operator-cloud-credent
 - arn:aws:iam::123456789012:role/<cluster>-a8m7-openshift-image-registry-installer-cloud-credenti
 - arn:aws:iam::123456789012:role/<cluster>-a8m7-openshift-ingress-operator-cloud-credentials
 - arn:aws:iam::123456789012:role/<cluster>-a8m7-openshift-cluster-csi-drivers-ebs-cloud-credentia
OIDC Endpoint URL:          https://rh-oidc.s3.us-east-1.amazonaws.com/<cluster-id>

```

Reply:
We typically avoid using pipe commands into formatting commands that produce snippets from a command output. In my view, the calls outs provide guidance on how to find the relevant items seems ok. I realise that the output is long, but I think the complete reference sample is acceptable in this case.